app/src/main/java/com/kyphipraniti/fitnesstasks/adapters/RVAdapter.java

```diff
@@ -0,0 +1,73 @@
+package com.kyphipraniti.fitnesstasks.adapters;
+
+import java.util.ArrayList;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.kyphipraniti.fitnesstasks.R;
+import com.kyphipraniti.fitnesstasks.activities.WorkoutDetailActivity;
+
+/**
+ * Created by praniti on 10/11/17.
+ */
+
+public class RVAdapter extends RecyclerView.Adapter<RVAdapter.MyViewHolder> {
+
+    ArrayList<String> workoutNames;
+    ArrayList<Integer> workoutImages;
```

Comment:
these two array lists should probably be a single array list of `Task` objects yeah?

Reply:
We can keep this for now, and change it when we integrate.